jarlotte-mojo/src/main/scala/de/metacoder/jarlotte/JarPackagingMojoExecutor.scala

```diff
@@ -34,17 +34,20 @@ class JarPackagingMojoExecutor(mojo: JarPackagingMojo) {
 
     val targetDir = Paths.get(mojo.getProject.getBuild.getDirectory).toFile
     val projectFinalName: String = mojo.getProject.getBuild.getFinalName
-    val buildDir = Paths.get(mojo.getProject.getBuild.getDirectory, projectFinalName).toFile
+    val warFile = Paths.get(mojo.getProject.getBuild.getDirectory, s"$projectFinalName.war").toFile
+    val extractedWarDir = Paths.get(mojo.getProject.getBuild.getDirectory, "extracted-war", projectFinalName)
```

Comment:
Jap, kann ich machen. Die Frage ist, ob man nicht sowieso einen "jarlotte"- oder "de.metacoder.jarlotte"-Ordner anlegt und dort auch den "jarlotte-loader-extracted" hinlegt, so dass alles in diesem Unter-Ordner liegt.

Soll ich es für diesen PR auf "extracted-war-jarlotte" ändern?